modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/facet/SearchFacetRegistryImpl.java

```diff
@@ -28,8 +28,8 @@
 /**
  * @author Eudaldo Alonso
  */
-@Component(immediate = true, service = SearchFacetTracker.class)
-public class SearchFacetTracker {
+@Component(immediate = true, service = SearchFacetRegistryImpl.class)
+public class SearchFacetRegistryImpl {
 
 	public List<SearchFacet> getSearchFacets() {
 		return _searchFacets;
```

Comment:
@JanisZhang, please rename it for SearchFacetRegistry, other case also named "*Registry". For example, LayoutStructureItemImporterRegistry. Impl should implement one interface.